### ISSUE TITLE
Missing medals

### ISSUE THREAD

**Issue by Axroy** [auduchinok/hwproj#39]
Sometimes some medals aren't assigned or maybe aren't displayed. See [http://hwproj.herokuapp.com/courses/1/terms/3](url), homeworks 4 and 5.

**Comment by auduchinok** [comment]
Yep, there can be trouble if the teacher remove `accepted` status after medals were issued to a student.
There should be more checks for such situations in `HomeworksController` and `TasksController`, I guess.
